src/components/snew/Content.js

```diff
@@ -97,12 +102,20 @@ export const CustomContent = ({
         </div>
       )}
       {isCMS ? (
-        <InvoiceFilter
-          header={header}
-          handleChangeFilterValue={onChangeFilter}
-          filterValue={filterValue}
-          invoiceCounts={invoiceCounts}
-        />
+        <div>
```

Comment:
Consider using a React fragment here to avoid adding useless tags:
```suggestion
         <>
          <InvoiceFilter
            header={header}
            handleChangeFilterValue={onChangeFilter}
            filterValue={filterValue}
            invoiceCounts={invoiceCounts}
          />
          <DateFilter
            header={header}
            handleChangeDateFilter={onChangeDateFilter}
            monthFilterValue={monthFilterValue}
            yearFilterValue={yearFilterValue}
          />
        </>
```